build.ts

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env -S deno run --allow-read --allow-write --allow-env
 import {
-  AlphaColor,
-  Color,
-  Labels,
-  variants,
-} from "npm:@catppuccin/palette@0.1.5";
+  CatppuccinFlavor,
+  flavorEntries,
+  flavors,
+} from "https://deno.land/x/catppuccin@v1.7.1/mod.ts";
```

Comment:
```suggestion
} from "https://deno.land/x/catppuccin@v1.7.1/mod.ts";
```
Let's use npm/JSR, deno.land/x is basically abandoned/deprecated at this point.